src/fsharp/TextLayoutRender.fs

```diff
@@ -167,8 +167,7 @@ module LayoutRender =
 
     let emitL f layout = renderL (taggedTextListR f) layout |> ignore
 
-    let toImmutableArray layout = 
```

Comment:
why this change?

Reply:
ImmutableArray is not in the default reference set for F# Interactive (i.e. the default reference set for the F# scripting model) and in my testing this is causing problems when using the API (though I could just reference it in my test scripts)

We can still use ImmutableArray but I just wanted to do it all-at-once intentionally (replacing all uses of `IList<_>` by `ImmutableArray<_>` for example) and also remembering to fix this problem.

I'll make a note of this at the top of this PR